src/realizations/catchment/Bmi_Fortran_Formulation.cpp

```diff
@@ -54,37 +54,37 @@ double Bmi_Fortran_Formulation::get_var_value_as_double(const int &index, const
     //  don't fit or might convert inappropriately
     std::string type = get_bmi_model()->GetVarType(var_name);
     if (type == "long double")
-        return (double) (get_bmi_model()->GetValue<long double>(var_name))[index];
+        return (double) GetValue<long double>(*get_bmi_model(), var_name)[index];
```

Comment:
Two small things:
1. Can we put the extra parentheses back here to match the other lines (the difference looks deliberate but I think it's not...or remove the other lines' extra parens optionally).
2. In this file (this line and those below) can we use `models::bmi::GetValue` instead of just `GetValue` to avoid ambiguity--GetValue is used elsewhere (bmi-cxx interface at least) and is a very ambiguous name so it would be good to be explicit about the namespace.